yarn.lock

```diff
@@ -2804,116 +3132,140 @@
+  dependencies:
+    "@sentry-internal/browser-utils" "8.0.0-beta.5"
+    "@sentry-internal/feedback" "8.0.0-beta.5"
+    "@sentry-internal/replay" "8.0.0-beta.5"
+    "@sentry-internal/replay-canvas" "8.0.0-beta.5"
+    "@sentry/core" "8.0.0-beta.5"
+    "@sentry/types" "8.0.0-beta.5"
+    "@sentry/utils" "8.0.0-beta.5"
+
+"@sentry/core@8.0.0-beta.5", "@sentry/core@^8.0.0-beta.5":
+  version "8.0.0-beta.5"
+  resolved "https://registry.yarnpkg.com/@sentry/core/-/core-8.0.0-beta.5.tgz#b68956a863c60af20a0a16ca83885124d6867cf5"
+  integrity sha512-DVPTRDMGAZXQY5AxQ3j24CnO0KKHQCZT7IP2ZtN4bVQd685inpSKNhVjNCOssI7zhV9QuW62mqbksJ2+JbBdPA==
+  dependencies:
+    "@sentry/types" "8.0.0-beta.5"
+    "@sentry/utils" "8.0.0-beta.5"
 
 "@sentry/jest-environment@^4.0.0":
   version "4.0.0"
   resolved "https://registry.yarnpkg.com/@sentry/jest-environment/-/jest-environment-4.0.0.tgz#037844bed70c8f13259ee01ab65ff8d36aef0209"
   integrity sha512-91jLBS8KbX2Ng0aDSP7kdE9sjiLc4qjp/jczTbmvOvuHxoaQ9hSLaEpsthnnUQ/zNeprZMkOC9xlS+zABw3Zmw==
 
-"@sentry/node@^7.111.0":
-  version "7.111.0"
-  resolved "https://registry.yarnpkg.com/@sentry/node/-/node-7.111.0.tgz#b18fddef2b38e0d9632a1e9db1f4667a2ebe9e2b"
-  integrity sha512-bTLZNETT7W89HEk04rwsch02KSpu++Yec/BEyM3AxUNY+ZQ9ZLL/lrNZuCwbe7fURpKoZrvGAhxpPjgs5UcB9w==
-  dependencies:
-    "@sentry-internal/tracing" "7.111.0"
-    "@sentry/core" "7.111.0"
-    "@sentry/types" "7.111.0"
-    "@sentry/utils" "7.111.0"
-
-"@sentry/profiling-node@^7.109.0":
-  version "7.109.0"
-  resolved "https://registry.yarnpkg.com/@sentry/profiling-node/-/profiling-node-7.109.0.tgz#fb34be1ec533b20ae37b7306a25b0fec1dccafbd"
-  integrity sha512-wdDxJ86/A71QTpbbn4LaiB6vmvGd3QEq7UOdZS/y/3SwV66zpkvNB8yPER41BgqDjqcmIY/va9fNi319iTREFw==
-  dependencies:
+"@sentry/node@8.0.0-beta.5", "@sentry/node@^8.0.0-beta.5":
+  version "8.0.0-beta.5"
+  resolved "https://registry.yarnpkg.com/@sentry/node/-/node-8.0.0-beta.5.tgz#a986672e23ab2694aecd562148d0b6fd6da2d520"
+  integrity sha512-eYEcQQy/ra7FOXYJRuUZJq+EddXY+E4mNqcAWY+VM45QFtzpfdlMOG8KjpN4Z+ptoeWj5cquao1JZSMWo4cOEw==
+  dependencies:
+    "@opentelemetry/api" "^1.8.0"
+    "@opentelemetry/context-async-hooks" "^1.23.0"
+    "@opentelemetry/core" "^1.23.0"
+    "@opentelemetry/instrumentation" "0.48.0"
+    "@opentelemetry/instrumentation-connect" "0.35.0"
+    "@opentelemetry/instrumentation-express" "0.35.0"
+    "@opentelemetry/instrumentation-fastify" "0.35.0"
+    "@opentelemetry/instrumentation-graphql" "0.39.0"
+    "@opentelemetry/instrumentation-hapi" "0.36.0"
+    "@opentelemetry/instrumentation-http" "0.48.0"
+    "@opentelemetry/instrumentation-koa" "0.39.0"
+    "@opentelemetry/instrumentation-mongodb" "0.39.0"
+    "@opentelemetry/instrumentation-mongoose" "0.37.0"
+    "@opentelemetry/instrumentation-mysql" "0.37.0"
+    "@opentelemetry/instrumentation-mysql2" "0.37.0"
+    "@opentelemetry/instrumentation-nestjs-core" "0.36.0"
+    "@opentelemetry/instrumentation-pg" "0.40.0"
```

Comment:
Would it be possible to make these something more like peer dependencies?

Is there an ongoing discussion anywhere about this? I would expect people (our customers) to be unhappy about how many new things we're pulling in

Reply:
This is basically the tradeoff we have for auto-instrumentation with minimal setup. Without this, users would need to manually install a bunch of stuff 😬 so it's something we are aware of. If we find this to become a problem we'll look into this more.